guides/noncruncher-gridcoin-setup.htm

```diff
@@ -46,7 +46,7 @@ <h5>Step 3</h5>
                     <h4 class="text-bold">Gridcoin Introduction</h4>
                     <p>The following guides explain how to set up the Gridcoin Research wallet client on the platform of your choice.</p>
                     <p>
-                        If you want to run Gridcoin as a non-cruncher, you do not need to install the BOINC client. In this case, you will only earn POS interest and not receive research rewards due to a lack of BOINC research. The amount and frequency of interest depends on the quantity of coins you are holding in your wallet/client.
+                        If you want to run Gridcoin as a non-cruncher, you do not need to install the BOINC client. In this case, you will only earn staking rewards. The amount and frequency of interest depends on the quantity of coins you are holding in your wallet/client.

```

Comment:
It might be worth still clarifying that you don't get the research rewards since research rewards still can be something one gets from staking

Reply:
I am a bit confused by this suggestion, but I think the edits I just added should work, if not, can you clarify?

Reply:
Yeah, the edits you made fixed what I was trying to mention